webapp/channels/src/components/suggestion/emoticon_provider.tsx

```diff
@@ -131,7 +133,7 @@ export default class EmoticonProvider extends Provider {
 
                         // if the emoji has skin, only add those that match with the user selected skin.
                         if (emojiMatchesSkin(emoji, skintone)) {
-                            matchedArray.push({name: alias, emoji, type: Preferences.CATEGORY_EMOJI});
+                            matchedArray.push({name: alias, emoji, type: suggestionTypeEmoji});
```

Comment:
None of this code relates to preferences, so this was just a weird case where we used a constant that happened to have the value we wanted for an unintended purpose